contracts/interfaces/IRoleVoting.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
```

Comment:
question (blocking): Why the change from Apache-2 to MIT?

Have I missed an update about the licensing? 

_(MIT is close in permissiveness to Apache-2, but there are differences around protection of brand)_